test.cpp

```diff
@@ -2900,6 +2917,8 @@ int main(int argc, char **argv)
     TEST_CASE(tokenMacro4);
     TEST_CASE(tokenMacro5);
 
+    TEST_CASE(makeTokenListNoFile);
```

Comment:
hmm.. the test cases are ordered alphabetically more or less.. and I prefer to put related test cases together so somehow I would like to put this near some other test cases where we test the TokenList constructor..

I have the feeling that the "readfile_" tests are somewhat related. They test the raw parsing of a file I think. I suggest you change the name of this test to "readfile_file_not_found"